Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js

```diff
@@ -631,24 +626,6 @@ WI.NavigationSidebarPanel = class NavigationSidebarPanel extends WI.SidebarPanel
             this._updateContentOverflowShadowVisibilityDebouncer.delayForTime(0);
     }
 
-    _checkForStaleResourcesIfNeeded()
-    {
-        if (!this._checkForStaleResourcesTimeoutIdentifier || !this._shouldAutoPruneStaleTopLevelResourceTreeElements)
-            return;
-        this.pruneStaleResourceTreeElements();
-    }
-
-    _checkForStaleResources(event)
-    {
-        console.assert(this._shouldAutoPruneStaleTopLevelResourceTreeElements);
-
-        if (this._checkForStaleResourcesTimeoutIdentifier)
-            return;
-
-        // Check on a delay to coalesce multiple calls to _checkForStaleResources.
-        this._checkForStaleResourcesTimeoutIdentifier = setTimeout(this.pruneStaleResourceTreeElements.bind(this));
-    }
-
     _isTreeElementWithoutRepresentedObject(treeElement)
     {
         return treeElement instanceof WI.FolderTreeElement
```

Comment:
i just noticed this function, and i think a lot of the things in this are no longer valid (e.g. `WI.EventBreakpointTreeElement` does have a `representedObject`), so id suggest looking into this a bit more as an unrelated followup

Reply:
Filed at https://bugs.webkit.org/show_bug.cgi?id=274915